server/controllers/chartsController/helpers.js

```diff
@@ -432,12 +432,17 @@ export const sortTableRowDataBySite = (dataBySite) => {
       site.targets[TOTAL_LABEL] = targetTotal
       return site
     })
-    .sort(sortAllSitesBeforeTotalsSite)
+    .sort(sortSitesByNameAndTotalsAtBottom)
 }
 
-const sortAllSitesBeforeTotalsSite = (siteA, siteB) => {
+const sortSitesByNameAndTotalsAtBottom = (siteA, siteB) => {
   if (siteA.name === TOTALS_STUDY) return 1
-  if (siteB.name === TOTALS_STUDY) return 1
+  if (siteB.name === TOTALS_STUDY) return -1
+
+  if (siteA.name === siteB.name) return 0
```

Comment:
do we need this check? The default is to return 0 so we could let this pass through to there.